fast64_internal/oot/c_writer/oot_level_c.py

```diff
@@ -195,18 +195,23 @@ def ootRoomMeshToC(room, textureExportSettings):
 		raise PluginError("Error: Room " + str(room.index) + " has no mesh children.")
 
 	meshHeader = CData()
-	meshHeader.header = "extern MeshHeader" + str(mesh.meshType) + " " + mesh.headerName() + ';\n'
-	meshHeader.source = "MeshHeader" + str(mesh.meshType) + " " + mesh.headerName() + ' = ' +\
-		"{ {" + str(mesh.meshType) + "}, " + str(len(mesh.meshEntries)) + ", " +\
-		"(u32)&" + mesh.entriesName() + ", (u32)&(" + mesh.entriesName() + ") + " +\
-		"sizeof(" + mesh.entriesName() + ") };\n\n"
+	meshHeader.header = "extern MeshHeader" + mesh.meshType + " " + mesh.headerName() + ";\n"
+	meshHeader.source = '\n'.join((
+		"MeshHeader" + mesh.meshType + " " + mesh.headerName() + "= {",
+		indent + mesh.meshType + ",",
+		indent + str(len(mesh.meshEntries)) + ",",
+		indent + mesh.entriesName() + ",",
+		indent + mesh.entriesName() + " + sizeof(" + mesh.entriesName() + ")",
```

Comment:
```suggestion
		indent + mesh.entriesName() + " + ARRAY_COUNT(" + mesh.entriesName() + ")",
```
with pointer arithmetics, `meshEntries + sizeof(meshEntries)` would be `(u32)meshEntries + sizeof(MeshHeaderEntry)*sizeof(meshEntries)` instead of `(u32)meshEntries + sizeof(meshEntries)`

array_count if it works would be the expected behavior

note this isn't critical as the entries end is not used by the game but may as well get it right

Reply:
oh yea, I haven't thought about using array_count there